src/App.js

```diff
@@ -174,11 +174,19 @@ class App extends React.Component {
               </svg></li>
             <li>PKey: {PrivateKey}</li>
             <li>Chain Id: {chainId}</li>
-            <li>RSK Endpoint: {rskEndpoint}</li>
-            <li>Hub Endpoint: {hubEndpoint}</li>
+            <li>RSK Endpoint: <a href={rskEndpoint}>{rskEndpoint}</a></li>
+            <li>Hub Endpoint: <a href={hubEndpoint}>{hubEndpoint}</a></li>
             <li>Registry Contract Address: {registryAddress}</li>
             <li>Token Network Addresses: {tokenNetworkAddresses.join(', ')}</li>
-            <li>Notifier Endpoints: {notifierEndpoints.join(', ')}</li>
+            <li>
+              Notifier Endpoints:&nbsp;
+              {notifierEndpoints.map((notifierEndpoint, index) => {
```

Comment:
not sure if this fails if we start the dapp without notifiers (thats possible)

Reply:
That will never fail since the function that maps the environment variable controls that and returns an empty array so we should be ok.

Reply:
We could anyway add a control that if array is empty this is removed, i will do it quickly